README.md

```diff
@@ -13,17 +13,40 @@ Check out the `examples` folder for detailed descriptions and code examples for:
 
 # Installation
 
-Phoenics can be installed via pip
+## Environment (Optional)
+
+It is suggested to have a fresh environment in order to avoid to have pip modules compatibility issues. This is optional but recommended.
```

Comment:
We suggest to create a new environment for the installation of Phoenics to avoid any compatibility issues with pre-existing python packages. This is optional, but recommended

Reply:
Done